site/content/internal/mobile-build-process/bump-build-number.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Bump Build Number
+date: 2017-11-07T14:28:35-05:00
+subsection: Mobile Build Process
+weight: 2
+---
+
+This must be done in your local copy of the [mattermost-mobile](https://github.com/mattermost/mattermost-mobile)
+
+1. Source the environment variables
+    ```
+    export LC_ALL="en_US.UTF-8"
+    
+    ############ MATTERMOST BUILD ############
+    export COMMIT_CHANGES_TO_GIT=true
+    export BRANCH_TO_BUILD=master
+    export GIT_LOCAL_BRANCH=build-number
+    export RESET_GIT_BRANCH=false
+    
+    export INCREMENT_BUILD_NUMBER=true
+    export INCREMENT_BUILD_NUMBER_MESSAGE="Bump app build number to"
+    ```
+    
+    {{%note "Env vars"%}}Alternatively you can copy the environment variables to the `mattermost-mobile/fastlane/.env` file.{{%/note%}}
+    
+    {{%note "Specify build number"%}}Sometimes you need to specify the build number instead of just increasing it by one.
```

Comment:
@migbot it will depend on the case, I don’t think there is a defined use case for it, perhpas a dot release will need it, perhaps not, again it depends on the use case. We have the ability to set it manually that is why it is documented.